lib/script_base.rb

```diff
@@ -56,7 +56,7 @@ def run
     stderr.puts "*Task*: `#{result.subtask}`"
     stderr.puts "*UUIDs*: #{result.uuids.map { |uuid| "`#{uuid}`" }.join(', ')}"
     if result.messages.present?
-      stderr.puts "*Messages*: #{result.messages.map { |message| "* `#{message}`" }.join('\n')}"
+      stderr.puts "*Messages*: #{result.messages.map { |message| "\n* #{message}" }}"
```

Comment:
```suggestion
      stderr.puts "*Messages*:\n#{result.messages.map { |message| "* #{message}" }.join("\n")}"
```

Reply:
we definitely do not want single-quote `'\n'` because that isn't a control character:
```
irb(main):001:0> puts '\n'
\n
=> nil
irb(main):002:0> puts "\n"

=> nil
```

Reply:
Got it!  Done